### PR TITLE
[team_hello_1]: Add List Note.

### PR DESCRIPTION
**Pull request by JJackMoO**
### What?

- brief: 添加了源码中List部分的测试代码与学习笔记
- author: Rookie-kai, JackMo

### What changes are included in this PR?

对team _hello 下的 rookie 个人文件夹下的main.cpp代码进行了修改，并添加了List的Markdown笔记文件
对team _hello 下的 jackmo 个人文件夹下的main.cpp代码进行了修改，并添加了List的Markdown笔记文件

### Are these changes tested?

源码见 team_hello/rookie/src/task1/main.cpp 和 team_hello/jackmo/src/task1/main.cpp，里面完成了list测试程序

### CODE REVIEW

File: stl/team_hello/jackmo/src/task_1/main.cpp
Comment (Light-City) [inline stl/team_hello/jackmo/src/task_1/main.cpp:1]: ?? just helloworld?

File: stl/team_hello/rookie/src/task_1/main.cpp [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#include <iostream>
+#include <list>
+#include <algorithm>
+
+int main() {
+    std::list<int> List;
```
Comment (Light-City) [inline stl/team_hello/rookie/src/task_1/main.cpp:6]: lowercase

File: stl/team_hello/tracehere/src/task_1/main.cpp
Comment (Light-City) [inline stl/team_hello/tracehere/src/task_1/main.cpp:1]: no test.

File: stl/team_hello/jackmo/src/task_1/main.cpp
Comment (JJackMoO) [reply stl/team_hello/jackmo/src/task_1/main.cpp:1]: The notes and code are still being improved

File: stl/team_hello/tracehere/README.md
Comment (Light-City) [inline stl/team_hello/tracehere/README.md:1]: why empty?

File: stl/team_hello/tracehere/src/task_1/main.cpp [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#include <iostream>
```
Comment (Light-City) [inline stl/team_hello/tracehere/src/task_1/main.cpp:1]: this is a hello world test, but for now

**Comment by Light-City** [comment]
Time has expired, eliminated, refused to merge

**Pull request closed without merging**